### ISSUE TITLE
Presentation

### ISSUE THREAD

**Issue by medensqui** [PlanHubMe/PlanHub#172]
Do we want to have some sort of presentation at Assembly?

Or for the club?

Infogr.am is really good for infographics if we need them.
Google Slides or Keynote or PowerPoint or Prezi or any presentation software would suffice for the actual presentation
Paging @thatoddmailbox @nywillb @willweinstein

**Comment by willbarkoff** [comment]
@thatoddmailbox Canva is good for posters and social media announcements.

**Comment by thatoddmailbox** [comment]
@medensqui Idk how it works in ms but I know for hs assembly announcements there is a place in first class to put them. Or maybe talk to that person you emailed for clubs.

**Comment by willweinstein** [comment]
@thatoddmailbox as long as i don't have to get on the stage, do whatever you want

**Comment by thatoddmailbox** [comment]
@medensqu Actually we might want to wait until we have My Day ready so we can talk about sleep.

**Comment by medensqui** [comment]
@thatoddmailbox Yeah, you have to fill out a gdocs form. But remember we aren't a club yet...

**Comment by thatoddmailbox** [comment]
@medensqui 

> But remember we aren't a club yet...

WELL WE SHOULD BE

**Comment by medensqui** [comment]
@thatoddmailbox I KNOW BUT WE CANT UNTIL LECHICH REPLIES

**Comment by medensqui** [comment]
@thatoddmailbox AND WE HAVE A FACULTY ADVISOR

**Comment by thatoddmailbox** [comment]
@medensqui

> LECHICH REPLIES

WHEN DO YOU HAVE LAB CAN WE GO ANNOY HIM

> FACULTY ADVISOR

SSSSHHHHH

**Comment by medensqui** [comment]
@thatoddmailbox LAST PERIOD? OR SECOND HALF INTERVAL?

**Comment by thatoddmailbox** [comment]
@medensqui I CANNOT DO LAST PERIOD BECAUSE I HAVE FRENCH. HOPEFULLY HE DOES NOT EAT LUNCH SECOND HALF OF INTERVAL.

**Comment by willweinstein** [comment]
@medensqui @thatoddmailbox who's the faculty advisor

**Comment by thatoddmailbox** [comment]
@willweinstein we are still deciding on compsci teacher or new lab.

**Comment by medensqui** [comment]
@thatoddmailbox So
lemme show you pros and cons

MSW 
- Can help us w/ code
- We get auth privliges
- We get chargers
- People like us b/c we're connected to CS
- Possibly get CS credit? (not sure)

Neiers
- We can bug him
- Umm...Not much else

**Comment by medensqui** [comment]
Btw the last two bullets are for Neiers @thatoddmailbox

**Comment by thatoddmailbox** [comment]
@medensqui GO ON SKYPE

**Comment by medensqui** [comment]
@thatoddmailbox OK IM IN MATH BUT IM WHERE IM SUPPOSED TO BE FOR THE END OF TODAY SO OK

**Comment by amazansky** [comment]
I am working on a Prezi. If you want to see/work on it give me your Prezi username in Slack. Also if you haven't already, sign up for a prezi education account to get a free plan.

**Comment by amazansky** [comment]
wait only one person can be assigned? oh well, @medensqui you are still "assigned" in my mind... but I _am_ working on the prezi.

**Comment by willbarkoff** [comment]
I want to work on prezi too, I sent u my prezi username so can I edit

**Comment by amazansky** [comment]
Okay (way too late lol)

**Comment by willbarkoff** [comment]
@thatoddmailbox 

> infogr.am is really good for infographics
> Adobe post is good for things that don't need much detail: [iOS](http://c00.adobe.com/9c5f8efbfd205eb5ababce6d862135839de6e5ed55a3d0034475b7bee0ff552c/x46gwkfi/i/1051937863) [more info](http://post.adobe.com)

Edited by @RobotXLabs to add http to the beginning of the link so it didn't 404
